cloud-aws-cloudformation/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/AwsNetworkConnector.java

```diff
@@ -136,6 +136,7 @@ public NetworkCidr getNetworkCidr(Network network, CloudCredential credential) {
                 List<String> cidrs = vpc.getCidrBlockAssociationSet()
                         .stream()
                         .map(VpcCidrBlockAssociation::getCidrBlock)
+                        .distinct()
                         .collect(Collectors.toList());
                 LOGGER.info("The VPC {} CIDRs block are {}.", vpc.getVpcId(), cidrs);
                 vpcCidrs.addAll(cidrs);
```

Comment:
do we need a distinct on the whole list?

Reply:
Where should we need distinct? I cannot distinct an element

Reply:
Ok, As we talked, I will distinct the vpcCidrs. Good catch